### ISSUE TITLE
“Uh Oh! Looks like something went wrong.”

### ISSUE THREAD

**Issue by Xarcaneali** [OddDevelopment/OperaGX-Nitro-Gen#26]
Regardless of the steps I follow, I consistently receive an error message stating, "Uh Oh! Looks like something went wrong."

To ensure that the problem was not account-specific, I have tried to claim Nitro on two different accounts: one that is 3 years old and another that I recently created. It is worth noting that neither account has previously claimed or used Nitro. Unfortunately, the issue persists across both accounts, and I am unable to successfully claim Nitro.

i would greatly appreciate any guidance or solutions you can provide.

I look forward to your prompt response and hope for a favorable resolution.

**Comment by EnoughForYou** [comment]
same dude

**Comment by Lomusire** [comment]
Opera made login required and changed their api endpoint. However this still easaly can be bypassed by using a session token since they don't seem to ratelimit how many tokens can be generated per session token. Here my own updated unoptomized but working version wich I made whithin 5 min feel free to improve it / use it as you want 
https://replit.com/@Lomusire/opera-new-gggpt4#main.py

**Comment by Xarcaneali** [comment]
@Lomusire 

I must admit, I'm not a coding expert and, funny enough, I haven't managed to get a single piece of code to work yet, lol. I even tried running the code you shared, but no luck—it's acting up just like before, and the issue persists. I'm wondering if I might be missing a step. Is there something else I should do besides running the code on Replit using Chrome? Any guidance would be greatly appreciated!

**Comment by Lomusire** [comment]
Does it generate promocode? If yes it might be due to browser cookies or ip try clearing browser data / different browser and use mobile data (turn on and off airplane mode) to switch ip adress

**Comment by Xarcaneali** [comment]
Yes, it's generating the links perfectly; the only issue is they aren't working. However, I'll try out the solutions you mentioned and let you know if they resolve the problem. Thank you for your assistance!

**Comment by Xarcaneali** [comment]
@Lomusire 
Yes, it's generating the links perfectly; the only issue is they aren't working. However, I'll try out the solutions you mentioned and let you know if they resolve the problem. Could you also recommend which browser I should use for this? Thank you for your assistance!

**Comment by Lomusire** [comment]
I recommend any chromium based Browser. Does the script on this repo also still generate promocode for you? Thought it stopped working whith the method odd used that's why I shortly created the new updated script

**Comment by Lomusire** [comment]
Does the promo link still show and the problem occurs after redemption if yes it's cause discord limits how often the same card can be used for redemption you need to get vcc cards

**Comment by Xarcaneali** [comment]
Yes, when I click on the link, it opens, but once I click on "Next," the message "Something went wrong" pops up. So far, I haven't been able to claim even once, let alone use my card. Basically, I don't even reach that option.
![Screenshot (305)](https://github.com/OddDevelopment/OperaGX-Nitro-Gen/assets/161977309/79629446-bfc8-46a4-bd38-59353e241f99)
![Screenshot (306)](https://github.com/OddDevelopment/OperaGX-Nitro-Gen/assets/161977309/5309200b-11e4-47ba-bb41-f0328198d40f)

**Comment by Xarcaneali** [comment]
@Lomusire 

Also, I'm unclear about which promo codes you're referring to. Could you please add a picture as an example so I could understand better?

**Comment by Xarcaneali** [comment]
@Lomusire 

I switched to Brave and even tried Edge while using data connection from my phone and have been toggling airplane mode on and off. However, whenever I click on the generated links, it still says "Uh oh, something went wrong." 

Sorry to bother you, but are there any other solutions you could recommend?

**Comment by Lomusire** [comment]
Oh I'm so sry seems like you're actually right and I was missing something

**Comment by Lomusire** [comment]
Seems like my aproch of token generation is not working at all confused the network requests....

**Comment by Xarcaneali** [comment]
No worries at all! Thanks for looking into it.. Is there a way we can troubleshoot or fix the issue with the token generation? I'm open to any suggestions or solutions you might have.

**Comment by Xarcaneali** [comment]
No worries at all! Thanks for looking into it.. Is there a way you could troubleshoot or fix the issue with the token generation? I'm open to any suggestions or solutions you might have.
